readme.md

```diff
@@ -274,6 +276,35 @@ function usePick(): (...routes: Array<Route>) => Route
 
 Returns a function that lets you choose a route from an array of routes, or `null` if nothing matched. The selected route is found by traversing the parents of the picked route (`useLocationMatch`).
 
+---
+### `useIsCurrent`
+
+```js
+function useIsCurrent(route, params = {}): boolean
```

Comment:
Leest alsof params optioneel is, maar dat is het dus eigenlijk niet?

Reply:
Sommige routes hebben geen params